### ISSUE TITLE
Feature request: console level

### ISSUE THREAD

**Issue by furstenheim** [puppeteer/puppeteer#744]
Page offers a console event but it is not possible to know the level of the message (log, error...).

    page.on('console', function (...args) {
        // No way to know if this is an error
    })

Would it be possible to implement a way to know the level? I don't know what would be the cleanest way, either one more event per level:

    page.on('consoleError', ...)

or a new type of event:

    page.on('consoleWithType', function ({type, args}) ...

Basically here: https://github.com/GoogleChrome/puppeteer/blob/master/lib/Page.js#L308 the event type is totally ignored

**Comment by michaelfward** [comment]
@ebidel - I didn't know if creating a `Console` object with its own `Events` class property would be preferable; I couldn't find anywhere else in the project that didn't explicitly declare it's events, or emit more than one at a time. 

Let me know what you recommend and I'll submit a PR!

**Comment by aslushnikov** [comment]
@michaelfward I like `page.console` namespace that will deliver specific events; go for it!

```js
page.console.on('log', console.log);
page.console.on('warn', console.warn);
page.console.on('debug', console.debug);
page.console.on('pageerror', console.error);
```
